### ISSUE TITLE
Not compatible with FuckAdBlock 3.2.1

### ISSUE THREAD

**Issue by TRGReal** [Mechazawa/FuckFuckAdblock#29]
Welp, it's time. They've released a new version, and well? It's blocked FuckFuckAdBlock. Please update this to fix it and make sure that they cannot patch it again. Everyone hates ads, I'm sure you do too (you created it probably for this purpose)

**Comment by TRGReal** [comment]
Oops.. Sorry, I think Fuck FuckAdBlock still blocks 3.2.1

**Comment by ylluminarious** [comment]
+1

**Comment by sitexw** [comment]
+1

**Comment by AxDSan** [comment]
+1

**Comment by DaanBiesterbos** [comment]
Everyone loves to get stuff for free. It is funny how people miss the point of why they get stuff for free. Do not get me wrong, I am all for legislations, and getting rid of spam. Which is indirectly already happening because search engines are bitch slapping spammy websites down the search results. But within reason there should be some acceptable middle ground. Writing interesting content? Costs money. Marketing? Costs money.  Hiring developers and maintaining a website? Costs a lot money.  So considering all ads as spam is morally wrong. Consider them as a trade off for all a lot of stuff you get for free while browsing the internet. Because where do you think the revenue will come from if it not from advertisements? No company will spend money if there is nothing to gain. How would a world without ads look like? I guess waaaaay more articles that exist only to convince you to buy some product.  Now I prefer to ignore a few ads over wasting my time reading articles half way just to conclude that this is yet another empty article written to convince me to do a particular thing. You already see it happening.  At least some ad is easy to ignore.

My personal view on this? If you don' t like ads, than don' t use the website. Spammers are already being targeted heavily and luckely have no bright future on the web. But the websites you are enjoying are investing more than you realize in the stuff you are consuming for free. Yet you are not giving anything back. Claiming otherwise is naive and selfish. You don' t need to take my word for it. Everyone should realize that the ability to earn money is essential is this world. Very little comes for free. I challenge you to think about how the internet would work without ads.  Where will the money come from? 

I am glad this plugin does not work. Because websites should at least be able to measure how many people are not paying and are entitled to make business decisions based on numbers. Don' t forget, it is their website. Not yours. Is the website no longer free to use? Than don' t complain either. You are part of the reason why. 

I wish you all a wonderful day none the less.

**Comment by ylluminarious** [comment]
@DaanBiesterbos You're not wrong. Ads are helpful in making money for the vast majority. It is self-evident and no one is arguing against it. But I, as my own person, have a right to block any content on my computer that I do not want to see. Even if it's on someone else's website.

Sure, maybe that hurts the economy somewhat. Maybe it makes life a bit harder for some people. But maybe I don't want to see smut just because it makes someone money. Maybe I don't care about any products when I'm trying to work and earn a living myself. Maybe I don't like the invasive nature of most ads which collect undue information about me and my system.

If I am that way, I am totally within my rights. If I purchased a newspaper in real life and hired someone to tear out all the ads for me before I read it, that is also within my rights. Maybe it hurts some businesses. But I have a right to not see ads on my own purchased newspaper (or device).

If you don't like that, fine. You're also within your rights. But people who use adblockers are in the minority anyway. Most people see ads whether they like it or not. If I were on the receiving end of an adblocker, I wouldn't like it but I would also be sympathetic to them. At the end of the day, this is why people need to diversify their income sources. If you're so reliant on ads that it cripples your business when people block them, you need another way to make money with your service. If you can't do that, then you honestly need to rethink your business model. This doesn't just apply to ads anyway; this is just a generally good principle for businesses to follow.

I wish you a wonderful day as well.

**Comment by DaanBiesterbos** [comment]
I agree with you. And you are right. I completely agree that everybody has a right to choose whether they want to see ads. I am not saying I don' t like it when people don' t see ads. I am worried about the consequences, and there will be consequences when accurate information is no longer profitable.
There is always someone making money. Directly, indirectly it does not matter. What matters is the intent of the authors of that content. No employee is going to write about the competition because they know that their product is much better, perhaps cheaper as well.

I do truly believe that most do underestimate the long term results of this. I love freedom, and I love to get accurate, unbiased (as much as possible) information. I worked for large and small companies. Which is why I am worried. I love the whole content marketing idea. But it works becaus competition challenge other websites to deliver relevant and mostly accurate information. And from my angle, you are much better off reading a blog from some industry expert than relying on the SEO optimized bulk that large companies will give you. Rolling out of their highly optimized content factory at whatever department. Time is money. And even if the content is relevant and interesting. It is always written with a certain intent. They want -you- to do something with it. Revenue trumps accuracy. Leaving out important information is not lying. Plain and simple. Is there a better alternative? Or a free solution that would do the job? Nope. Generally you won' t hear about it from them.
It is the future of the internet I am worried about. The reliability of what you see and read. Nothing is free, although sometimes it looks that way. There are exceptions but in most cases..... Investments must lead to revenue or you have no business. So who is earning money if you don' t see ads?  
Who would invest large amounts of money in the website when there  is nothing to gain from providing accurate information.  And in the end, who would you trust more? A big organisation giving you their solution. Or a website that gives you more options and gives you relevant and unbiased information from other angles? Running a website and writing quality content is expensive. Very expensive. I am not talking about yet another wordpress website. I am talking about small and midsize companies. High traffic websites, serious server and software development costs and competent employees that work on websites that earn revenue by providing better information. What would happen to those companies?
Either their services won' t be free anymore, they would turn to affliate marketing impacting the subjects they write about or they will disappear don't you agree?  (or perhaps sold and turned into some marketing channel)

To me ads like are taxes. Nobody likes them. Some might get away with avoiding them. But if everybody would do it.................  So yeah hiding them is your right. But it is ignorant to expect companies to deliver high quality and unbiased information for free.  I highly doubt anyone "likes"  to see advertisements.

**Comment by Fifoo** [comment]
Github isn't a website for political discussion I think. But in this case: Advertisements are manipulative, they go in from an angle of "we need to manipulate people to buy our stuff". In an optimal situation people decide they need something and then look up where to best buy it either by research or by word to mouth. Ads treat people like dumb mindless animals, which is contradicting the individualist narrative of the whole free-society story. It's the perfect example of people saying the one thing and doing the exact opposite hoping that no large enough quantity of people are going to catch on. You want to compete in quality, not in how good you are at getting a herd of people to fall for your marketing.
The advertisement thought tries to foster mindless consumption instead of trying to progress reasonable autonomous premeditated purchases, it's all without shame or sense of responsibility. 

Just because Ads are pretty much the only way for small websites to run "for free" doesn't mean that it's fine, it just means that there is an urgent need to come up with an actual alternative. Aslong that is the case, I am fine with reserving my right to not look at flashy ads that try to make me wanna buy things I definitely don't need and definitely will never buy. (I might even decide to never buy something based just on the fact that they do internet advertising)
